docs/jwst/msaflagopen/description.rst

```diff
@@ -0,0 +1,31 @@
+
+Description
+===========
+
+Overview
+--------
+
+The msaflagopen correction flags pixels that are affected by msa shutters
+that are stuck in the open position.
+
+Background
+__________
+
+The correction is applicable to NIRSPEC IFU and MSA exposure types.
```

Comment:
Let's use the official spelling of the instrument acronym "NIRS**pec**" (i.e. camelcase)